### ISSUE TITLE
(CSV.Parser.SyntaxError) Unterminated escape sequence

### ISSUE THREAD

**Issue by yury-dymov** [beatrichartz/csv#52]
Hi,

I've recently exported data to CSV from Postgres, therefore I assume that there is nothing wrong with the data from RFC perspective.

I am using latest csv release 1.4.2 and facing error mentioned in subject during parsing.

Sample, which gives me error with CSV header attached

[sample.csv.zip](https://github.com/beatrichartz/csv/files/440644/sample.csv.zip)
```
id,question_id,user_id,year,locked,text,locale,inserted_at,updated_at,image_file_name,image_content_type,image_file_size,image_updated_at,comment_count
170,144,8,2015,f,"ООО...Неее... это не ко мне((
Но мое любимое вот это:
""Тихо-тихо ползи,
Улитка, по склону Фудзи,
Вверх, до самых высот.""",ru,2015-03-05 17:12:55,2015-03-05 17:12:55,,,,,0

```

P.s. I don't think that Russian text makes any difference, right? :)

**Comment by beatrichartz** [comment]
Thanks for raising this - Russian should indeed not make a difference :)

Should be fixed in 1.4.3, can you have a look?

**Comment by yury-dymov** [comment]
Hey, @beatrichartz, that was blazing fast! :)

Issue seems to be resolved. At least no errors during parsing. Thank you!
